core/src/test/java/org/apache/calcite/plan/RelWriterTest.java

```diff
@@ -883,7 +917,8 @@ private RelOptSchema getSchema(RelNode rel) {
    * Deserialize a relnode from the json string by {@link RelJsonReader},
    * and dump it to text format.
    */
-  private String deserializeAndDumpToTextFormat(RelOptSchema schema, String relJson) {
+  private String deserializeAndDumpToTextFormat(
```

Comment:
This function should be called as `deserializeAndDumpToFormat`?

Reply:
or maybe just `deserializeAndDump`. BTW the javadoc needs to be updated as well.

Reply:
Thanks for your careful review. I have revised the PR accordingly.